docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -406,6 +416,10 @@ The following configuration set `Content-Type: application/vnd.android.package-a
 - `GRACEFUL_HAMMER_TIME`: **60s**: After a restart the parent process will stop accepting new connections and will allow requests to finish before stopping. Shutdown will be forced if it takes longer than this time.
 - `STARTUP_TIMEOUT`: **0**: Shutsdown the server if startup takes longer than the provided time. On Windows setting this sends a waithint to the SVC host to tell the SVC host startup may take some time. Please note startup is determined by the opening of the listeners - HTTP/HTTPS/SSH. Indexers may take longer to startup and can have their own timeouts.
 
+[^1]: Gitea's initial configuration form will not work correctly with
+`STATIC_URL_PREFIX` enabled.  Use this option only after successfully deploying
+Gitea and initializing the database.
+
```

Comment:
I think the initial configuration should work with STATIC_URL_PREFIX, the code is all the same.

And I have tested that if I set STATIC_URL_PREFIX correctly in `app.ini`, the installation page follows it.